src/mixins/filing-template-mixin.ts

```diff
@@ -302,7 +302,10 @@ export default class FilingTemplateMixin extends Mixins(AmalgamationMixin, DateM
     }
 
     // restore the Amalgamation Court Approval if it's True or False
-    if (draftFiling.amalgamationApplication.courtApproval !== null) {
+    if (
+      draftFiling.amalgamationApplication.courtApproval === true ||
```

Comment:
Can line 306 and 307 be just 1 line by doing if (draftFiling.amalgamationApplication.courtApproval)?

Reply:
No, because courtApproval could be undefined or null, and we don't want to set that in the store.